Simplenote/Information/NoteInformationController.swift

```diff
@@ -142,13 +142,18 @@ private extension NoteInformationController {
         }
 
         let referenceRows = references.map { (note) -> Row in
+            let date = DateFormatter.dateFormatter.string(from: note.modificationDate)
+            let value = String(references.count) +
+                " \(references.count > 1 ? "References" : "Reference")" +
+                ", Last Modified \(date)"
```

Comment:
This string needs to be localized, please check the following example: https://github.com/Automattic/simplenote-ios/blob/733fede9b6d5ebfc833384bbf775ff746e933be4/Simplenote/Classes/PinLock/PinLockBaseController.swift  

Using `> 1` condition is also not ideal because it doesn't work in all languages. That's something we need to discuss and improve later as a team. For now let's continue using `> 1` :-)

Reply:
That is a good way to do it!  Thanks for the suggestion.  Updated